# Docker image for RNA-Seq processes

The image is based on the [`public.ecr.aws/genialis/resolwebio/common`](
https://gallery.ecr.aws/genialis/resolwebio/common) image, derived from the
`ubuntu-20.04` version of [`public.ecr.aws/genialis/base`](
https://gallery.ecr.aws/genialis/resolwebio/base) image.

Included bioinformatics tools:
------------------------------
* BBMap (v37.90)
* Cufflinks (v2.2.1)
* Cutadapt (v2.4)
* DESeq2 (v1.30.0)
* DEXSeq (v1.36.0)
* dexseq_prepare_annotation2 (v1.0.0)
* edgeR (v3.32.1)
* gotea (v0.0.4)
* genometools (1.6.2)
* igvtools (v2.3.98)
* jbrowse (v1.16.6)
* loadSubread (0.0.1)
* regtools (0.5.0)
* rnanorm (1.3.0)
* RSEM (v1.4.0)
* rtracklayer (v1.50.0)
* Salmon (1.2.1)
* STAR (v2.7.10b)
* Trimmomatic (v0.36)
* tximport (v1.18.0)
* shRNAde (v1.0)
* UMI-tools (v1.0.0)
* Trim Galore (v0.6.6)
